app/external_doc.rb

```diff
@@ -119,7 +118,11 @@ def call
 
       doc.css('h1, h2, h3, h4, h5, h6').each do |node|
         text = node.text
-        id = StringToId.convert(text)
+        id = text
```

Comment:
This now duplicates the code in `DeveloperDocsRenderer`? What's the problem with this?